src/components/Personal-Zone/Searcher/FavoriteJobs/FavoriteJobs.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import AssignmentIndOutlinedIcon from '@material-ui/icons/AssignmentIndOutlined';
+import Button from '@material-ui/core/Button';
+import { Link } from 'react-router-dom';
+
+import styles from './FavoriteJobsStyle';
+
+const FavoriteJobs: React.FC<{}> = (): JSX.Element => {
+    const classes = styles();
+
+    return (
+        <div className={classes.root}>
+            <AssignmentIndOutlinedIcon style={{ fontSize: 100 }} />
+            <span className={classes.content}>
+                עדיין לא התעניינת באף תפקיד,<br/>
```

Comment:
Why do we use `<br/>` rather than handling it at jss level?

Reply:
why not